datasets/stsb_mt_sv/stsb_mt_sv.py

```diff
@@ -0,0 +1,93 @@
+# coding=utf-8
+# Copyright 2020 The TensorFlow Datasets Authors and the HuggingFace Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+import os
+
+import datasets
+
+
+_DOWNLOAD_URL = "https://raw.githubusercontent.com/timpal0l/sts-benchmark-swedish/master/data/stsb-mt-sv.zip"
+_TRAIN_FILE = "train-sv.tsv"
+_VAL_FILE = "dev-sv.tsv"
+_TEST_FILE = "test-sv.tsv"
+
+_CITATION = """\
+@article{isbister2020not,
+  title={Why Not Simply Translate? A First Swedish Evaluation Benchmark for Semantic Similarity},
+  author={Isbister, Tim and Sahlgren, Magnus},
+  journal={arXiv preprint arXiv:2009.03116},
+  year={2020}
+}
+"""
+
+_DESCRIPTION = "Machine translated Swedish version of the original STS-B (http://ixa2.si.ehu.eus/stswiki)"
+
+
+class SwedishTranslatedSTSB(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class StsbMtSv(datasets.GeneratorBasedBuilder):
```
The class name has to match the filename but in camelcase form